src/types/onyx/WalletTransfer.ts

```diff
@@ -0,0 +1,26 @@
+import CONST from '../../CONST';
+import * as OnyxCommon from './OnyxCommon';
+
+type WalletTransfer = {
+    /** Selected accountID for transfer */
+    selectedAccountID?: string;
```

Comment:
I think this may be both a number or a string based on `src/pages/settings/Payments/walletTransferPropTypes.js`